docs/ja/2-0-migration-guide.rst

```diff
@@ -0,0 +1,18 @@
+2.0 Migration Guide
```

Comment:
Could we get this english content removed? We generally only keep localized content in the `ja` directories. Providing users with potentially incorrect documentation is far worse than having users switch the language code in the URL or use a different tab with the english docs is.

If that proves to be challenging because of linking, I can take care of finding a solution to the incomplete translation that won't get outdated on my side. I have a half baked idea that involves using `make` to copy all the english content over if files do not exist :smiling_imp: